### ISSUE TITLE
nvgesture训练

### ISSUE THREAD

**Issue by huaxi-yui** [zhoubenjia/MotionRGBD-PAMI#2]
作者您好，感谢您出色的工作。我用您的代码从头开始训练nvgesture,训练效果一直不是很理想。请问能否提供一下您的预训练模型。

**Comment by zhoubenjia** [comment]
> 作者您好，感谢您出色的工作。我用您的代码从头开始训练nvgesture,训练效果一直不是很理想。请问能否提供一下您的预训练模型。

Hi, 感谢您对我们工作的关注，NvGesture问题请参考: [issues](https://github.com/damo-cv/MotionRGBD/issues/6)

**Comment by zhoubenjia** [comment]
> 作者您好，感谢您出色的工作。我用您的代码从头开始训练nvgesture,训练效果一直不是很理想。请问能否提供一下您的预训练模型。

hi! 请参考以下命令试试：
`CUDA_VISIBLE_DEVICES=0,1  python -m torch.distributed.launch --nproc_per_node=2 --master_port=1234 --use_env train.py --config config/NvGesture.yml --data /mnt/Data/datasets/Motion/nvgesture/ --splits /mnt/Data/datasets/Motion/nvgesture/dataset_splits/rgb/ --save ./output_dir/NV-TSM-M  --sample-duration 32 --opt sgd --lr 0.01 --sched cosine --smprob 0.0 --mixup 0.001 --shufflemix 0.2 --epochs 100 --distill 0.0 --type M --intar-fatcer 2 --finetune ./Checkpoints/NTU-RGBD-32-DTNV2-TSM/model_best.pth.tar --batch-size 8`

部分logs:
`
2024-02-02 05:49:23,098 Epoch: 98/100	Mini-Batch: 00001/00066	Lr: 4.9387066934183565e-05	Total_loss: 1.5663	CE_loss: 1.5663	Data_Time: 0.2971	Acc_s: 50.0000	Acc_m: 50.0000	Acc_l: 50.0000	Acc: 50.0000	Acc_top5: 50.0000
2024-02-02 05:49:29,759 Epoch: 98/100	Mini-Batch: 00011/00066	Lr: 4.9387066934183565e-05	Total_loss: 1.7241	CE_loss: 1.7241	Data_Time: 0.0270	Acc_s: 48.2955	Acc_m: 48.8636	Acc_l: 49.4318	Acc: 49.4318	Acc_top5: 53.4091
2024-02-02 05:49:36,330 Epoch: 98/100	Mini-Batch: 00021/00066	Lr: 4.9387066934183565e-05	Total_loss: 1.6484	CE_loss: 1.6484	Data_Time: 0.0142	Acc_s: 53.2738	Acc_m: 53.5714	Acc_l: 53.8690	Acc: 53.8690	Acc_top5: 58.0357
2024-02-02 05:49:42,720 Epoch: 98/100	Mini-Batch: 00031/00066	Lr: 4.9387066934183565e-05	Total_loss: 1.6217	CE_loss: 1.6217	Data_Time: 0.0096	Acc_s: 52.6210	Acc_m: 52.8226	Acc_l: 53.0242	Acc: 53.0242	Acc_top5: 56.6532
2024-02-02 05:49:49,141 Epoch: 98/100	Mini-Batch: 00041/00066	Lr: 4.9387066934183565e-05	Total_loss: 1.6075	CE_loss: 1.6075	Data_Time: 0.0073	Acc_s: 54.7256	Acc_m: 54.8780	Acc_l: 55.0305	Acc: 55.0305	Acc_top5: 58.9939
2024-02-02 05:49:55,768 Epoch: 98/100	Mini-Batch: 00051/00066	Lr: 4.9387066934183565e-05	Total_loss: 1.5993	CE_loss: 1.5993	Data_Time: 0.0059	Acc_s: 55.0245	Acc_m: 55.1471	Acc_l: 55.2696	Acc: 55.2696	Acc_top5: 58.9461
2024-02-02 05:50:02,741 Epoch: 98/100	Mini-Batch: 00061/00066	Lr: 4.9387066934183565e-05	Total_loss: 1.5937	CE_loss: 1.5937	Data_Time: 0.0049	Acc_s: 53.3811	Acc_m: 53.4836	Acc_l: 53.5861	Acc: 53.5861	Acc_top5: 57.6844
2024-02-02 05:50:07,783 Total_loss: 1.5913884748112073	CE_loss: 1.5913884748112073	Data_Time: 0.004536215554584156	Acc_s: 52.36742424242424	Acc_m: 52.46212121212121	Acc_l: 52.55681818181818	Acc: 52.55681818181818	Acc_top5: 56.628787878787875
2024-02-02 05:50:10,965 Epoch: 98	Mini-Batch: 0001/0021	Total_loss: 2.2750	Acc_sm: 79.1667	Acc_sl: 79.1667	Acc_lm: 83.3333	Acc_all: 83.3333	Acc_adaptive: 83.3333	Acc_adaptive_top5: 100.0000	Infer_Time: 0.0127
2024-02-02 05:50:16,723 Epoch: 98	Mini-Batch: 0011/0021	Total_loss: 2.0149	Acc_sm: 83.7121	Acc_sl: 83.3333	Acc_lm: 83.7121	Acc_all: 84.0909	Acc_adaptive: 83.3333	Acc_adaptive_top5: 97.3485	Infer_Time: 0.0112
2024-02-02 05:50:21,994 Epoch: 98	Mini-Batch: 0021/0021	Total_loss: 1.8489	Acc_sm: 85.5159	Acc_sl: 85.3175	Acc_lm: 85.5159	Acc_all: 85.7143	Acc_adaptive: 85.7143	Acc_adaptive_top5: 97.2222	Infer_Time: 0.0112
2024-02-02 05:50:25,993 Total_loss: 1.8685708045959473	Acc_sm: 85.03788063742898	Acc_sl: 85.4166689786044	Acc_lm: 85.60606280240145	Acc_all: 85.79545662619851	Acc_adaptive: 85.79545697298917	Acc_adaptive_top5: 96.78030430186878	Infer_Time: 0.011302786343025438
2024-02-02 05:50:25,994 valid_acc 85.7955, best_acc 86.9318
2024-02-02 05:50:28,892 Epoch: 99/100	Mini-Batch: 00001/00066	Lr: 3.216798680761542e-05	Total_loss: 1.5637	CE_loss: 1.5637	Data_Time: 0.2450	Acc_s: 100.0000	Acc_m: 100.0000	Acc_l: 100.0000	Acc: 100.0000	Acc_top5: 100.0000
2024-02-02 05:50:35,060 Epoch: 99/100	Mini-Batch: 00011/00066	Lr: 3.216798680761542e-05	Total_loss: 1.5655	CE_loss: 1.5655	Data_Time: 0.0223	Acc_s: 47.7273	Acc_m: 47.7273	Acc_l: 47.7273	Acc: 47.7273	Acc_top5: 52.2727
2024-02-02 05:50:41,431 Epoch: 99/100	Mini-Batch: 00021/00066	Lr: 3.216798680761542e-05	Total_loss: 1.5653	CE_loss: 1.5653	Data_Time: 0.0117	Acc_s: 57.1429	Acc_m: 57.1429	Acc_l: 57.1429	Acc: 57.1429	Acc_top5: 61.0119
2024-02-02 05:50:47,662 Epoch: 99/100	Mini-Batch: 00031/00066	Lr: 3.216798680761542e-05	Total_loss: 1.5654	CE_loss: 1.5654	Data_Time: 0.0080	Acc_s: 50.8065	Acc_m: 50.8065	Acc_l: 50.8065	Acc: 50.8065	Acc_top5: 54.8387
2024-02-02 05:50:54,164 Epoch: 99/100	Mini-Batch: 00041/00066	Lr: 3.216798680761542e-05	Total_loss: 1.5659	CE_loss: 1.5659	Data_Time: 0.0060	Acc_s: 46.3415	Acc_m: 46.3415	Acc_l: 46.3415	Acc: 46.3415	Acc_top5: 50.6098
2024-02-02 05:51:00,773 Epoch: 99/100	Mini-Batch: 00051/00066	Lr: 3.216798680761542e-05	Total_loss: 1.5881	CE_loss: 1.5881	Data_Time: 0.0049	Acc_s: 48.7745	Acc_m: 48.7745	Acc_l: 48.7745	Acc: 48.7745	Acc_top5: 52.9412
2024-02-02 05:51:07,934 Epoch: 99/100	Mini-Batch: 00061/00066	Lr: 3.216798680761542e-05	Total_loss: 1.5842	CE_loss: 1.5842	Data_Time: 0.0041	Acc_s: 51.8443	Acc_m: 51.8443	Acc_l: 51.8443	Acc: 51.8443	Acc_top5: 55.8402
2024-02-02 05:51:12,836 Total_loss: 1.5837028008518796	CE_loss: 1.5837028008518796	Data_Time: 0.0037556245471491957	Acc_s: 52.84090909090909	Acc_m: 52.84090909090909	Acc_l: 52.84090909090909	Acc: 52.84090909090909	Acc_top5: 56.91287878787879
2024-02-02 05:51:16,120 Epoch: 99	Mini-Batch: 0001/0021	Total_loss: 2.2765	Acc_sm: 79.1667	Acc_sl: 79.1667	Acc_lm: 83.3333	Acc_all: 83.3333	Acc_adaptive: 83.3333	Acc_adaptive_top5: 100.0000	Infer_Time: 0.0133
2024-02-02 05:51:21,603 Epoch: 99	Mini-Batch: 0011/0021	Total_loss: 2.0140	Acc_sm: 83.7121	Acc_sl: 83.3333	Acc_lm: 83.7121	Acc_all: 84.0909	Acc_adaptive: 83.7121	Acc_adaptive_top5: 97.7273	Infer_Time: 0.0113
2024-02-02 05:51:27,324 Epoch: 99	Mini-Batch: 0021/0021	Total_loss: 1.8482	Acc_sm: 85.5159	Acc_sl: 85.3175	Acc_lm: 85.5159	Acc_all: 85.7143	Acc_adaptive: 85.9127	Acc_adaptive_top5: 97.4206	Infer_Time: 0.0112
2024-02-02 05:51:31,270 Total_loss: 1.8679731217297642	Acc_sm: 85.03788063742898	Acc_sl: 85.4166689786044	Acc_lm: 85.60606280240145	Acc_all: 85.79545662619851	Acc_adaptive: 85.98485079678622	Acc_adaptive_top5: 96.96969812566584	Infer_Time: 0.011324741623618384
2024-02-02 05:51:31,270 valid_acc 85.9848, best_acc 86.9318
2024-02-02 05:51:34,459 Epoch: 100/100	Mini-Batch: 00001/00066	Lr: 1.9856491500783562e-05	Total_loss: 1.5629	CE_loss: 1.5629	Data_Time: 0.2822	Acc_s: 50.0000	Acc_m: 50.0000	Acc_l: 50.0000	Acc: 50.0000	Acc_top5: 56.2500
2024-02-02 05:51:40,692 Epoch: 100/100	Mini-Batch: 00011/00066	Lr: 1.9856491500783562e-05	Total_loss: 1.7138	CE_loss: 1.7138	Data_Time: 0.0257	Acc_s: 45.4545	Acc_m: 46.0227	Acc_l: 46.0227	Acc: 46.0227	Acc_top5: 50.5682
2024-02-02 05:51:46,981 Epoch: 100/100	Mini-Batch: 00021/00066	Lr: 1.9856491500783562e-05	Total_loss: 1.6431	CE_loss: 1.6431	Data_Time: 0.0135	Acc_s: 49.4048	Acc_m: 49.7024	Acc_l: 49.7024	Acc: 49.7024	Acc_top5: 53.2738
2024-02-02 05:51:53,251 Epoch: 100/100	Mini-Batch: 00031/00066	Lr: 1.9856491500783562e-05	Total_loss: 1.6221	CE_loss: 1.6221	Data_Time: 0.0091	Acc_s: 51.2097	Acc_m: 51.4113	Acc_l: 51.4113	Acc: 51.4113	Acc_top5: 55.6452
2024-02-02 05:51:59,582 Epoch: 100/100	Mini-Batch: 00041/00066	Lr: 1.9856491500783562e-05	Total_loss: 1.6080	CE_loss: 1.6080	Data_Time: 0.0069	Acc_s: 54.8780	Acc_m: 55.0305	Acc_l: 55.0305	Acc: 55.0305	Acc_top5: 59.1463
2024-02-02 05:52:06,152 Epoch: 100/100	Mini-Batch: 00051/00066	Lr: 1.9856491500783562e-05	Total_loss: 1.5995	CE_loss: 1.5995	Data_Time: 0.0056	Acc_s: 54.4118	Acc_m: 54.5343	Acc_l: 54.5343	Acc: 54.5343	Acc_top5: 58.5784
2024-02-02 05:52:13,084 Epoch: 100/100	Mini-Batch: 00061/00066	Lr: 1.9856491500783562e-05	Total_loss: 1.5940	CE_loss: 1.5940	Data_Time: 0.0047	Acc_s: 55.3279	Acc_m: 55.4303	Acc_l: 55.4303	Acc: 55.4303	Acc_top5: 59.5287
2024-02-02 05:52:17,959 Total_loss: 1.5917647661584797	CE_loss: 1.5917647661584797	Data_Time: 0.004315169923233263	Acc_s: 54.92424242424242	Acc_m: 55.01893939393939	Acc_l: 55.01893939393939	Acc: 55.01893939393939	Acc_top5: 59.18560606060606
2024-02-02 05:52:20,667 Epoch: 100	Mini-Batch: 0001/0021	Total_loss: 2.2771	Acc_sm: 79.1667	Acc_sl: 79.1667	Acc_lm: 83.3333	Acc_all: 83.3333	Acc_adaptive: 83.3333	Acc_adaptive_top5: 100.0000	Infer_Time: 0.0129
2024-02-02 05:52:26,343 Epoch: 100	Mini-Batch: 0011/0021	Total_loss: 2.0151	Acc_sm: 83.7121	Acc_sl: 83.3333	Acc_lm: 83.7121	Acc_all: 83.7121	Acc_adaptive: 83.3333	Acc_adaptive_top5: 97.7273	Infer_Time: 0.0113
2024-02-02 05:52:31,534 Epoch: 100	Mini-Batch: 0021/0021	Total_loss: 1.8499	Acc_sm: 85.7143	Acc_sl: 85.3175	Acc_lm: 85.5159	Acc_all: 85.5159	Acc_adaptive: 85.7143	Acc_adaptive_top5: 97.4206	Infer_Time: 0.0112
2024-02-02 05:52:35,701 Total_loss: 1.8695327585393733	Acc_sm: 85.22727446122603	Acc_sl: 85.4166689786044	Acc_lm: 85.60606280240145	Acc_all: 85.60606280240145	Acc_adaptive: 85.79545697298917	Acc_adaptive_top5: 96.96969812566584	Infer_Time: 0.011335601409276325
2024-02-02 05:52:35,702 valid_acc 85.7955, best_acc 86.9318
`
